src/AltinnCore/Common/Services/Implementation/AuthenticationAppSI.cs

```diff
@@ -45,30 +45,30 @@ public class AuthenticationAppSI : IAuthentication
         }
 
         /// <inheritdoc />
-        public async Task<HttpResponseMessage> RefreshToken()
+        public async Task<string> RefreshToken()
         {
             string endpointUrl = $"refresh";
             string token = JwtTokenUtil.GetTokenFromContext(_httpContextAccessor.HttpContext, Constants.General.RuntimeCookieName);
-            _logger.LogInformation($"Token from token utility{token}");
+            _logger.LogInformation($"Adding request header in api");
```

Comment:
Let's remove the logging that was added for debug purposes.

Reply:
same here